pycqed/analysis/measurement_analysis.py

```diff
@@ -4330,142 +4347,399 @@ def hanger_function_amplitude(x, f0, Q, Qe, A, theta):
 
 class Qubit_Spectroscopy_Analysis(MeasurementAnalysis):
 
+    """
```

Comment:
Not clear what the abbreviation GE is, I assume it is ground/excited and generally use lowercase for these, additionally the ef is in lowercase which is inconsistent (though I prefer lowercase).